### DISCUSSION TITLE
Route name

### DISCUSSION THREAD

**Discussion by Spriggan1992** [Milad-Akarie/auto_route_library#1517] [General]
In previous version of package, we have ability to specify a name of route like so: 
```
AutoRoute(
      name: 'LogInRouter',
      path: 'log_in',
      page: LogInScreen,
    )
```
 In current version we cannot specify the name of route. But If I need for example the same pages but with different name? 
Like this:
```
AutoRoute(
          name: 'CourseDetailsRouter',
          path: 'course_details',
          page: CourseDetailsScreen,
        ),
 AutoRoute(
          name: 'BookmarksCourseDetailsRouter',
          path: 'bookmarks_course_details',
          page: CourseDetailsScreen,
        ),
```
  Is it possible to get the same behavior in current version of package?

**Comment by Milad-Akarie** [comment]
@Spriggan1992 I'll be working on way to achieve this.
